frontend/src/entry/model/settings/AccessRequestSettings.tsx

```diff
@@ -23,6 +23,7 @@ export default function AccessRequestSettings({ model }: AccessRequestSettingsPr
     const updatedModelSettings = {
       settings: {
         ungovernedAccess: allowUngoverned,
+        allowTemplating: false,
```

Comment:
As we're not going to be changing this value in `AccessRequestSettings` it makes more sense to reuse the value already on the model. So I'd update this to `allowTemplating: model.settings.allowTemplating`

Reply:
Unresolved this as it's still outstanding